### ISSUE TITLE
Network-Diagnostics container is not health check enabled and logs are inaccessible when its host is powered OFF

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#9299]
**Rancher versions:**
rancher/server:v1.6.3

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**Ubuntu

**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**GCE

**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**Single node

**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**Cattle

**Steps to Reproduce:**
1. Deploy Network diagnostics catalog
2. Generate and access logs using
```
curl -X POST http://IP:8080/r/projects/1a90/network-diagnostics:8080/v1/logs
curl http://IP/r/projects/1a90/network-diagnostics:8080/v1/logs
```
3. Power OFF the host on which network diagnostics container is running. The network diagnostics service is not available.
```
soumyas-MBP:notes soumya$ curl http://IP:8080/r/projects/1a90/network-diagnostics:8080/v1/logs
Service unavailable
```

**Comment by leodotcloud** [comment]
Fixed by stack version: 0.0.2, image: rancher/network-diagnostics:v0.1.2

**Comment by soumyalj** [comment]
Tested with v1.6.8-rc1 using tool version 0.0.2. The network diagnostics container is healthcheck enabled. After a Power OFF, the container gets created on a different host and the network diagnostic service is available. POST and GET calls are successful
```
soumya$ curl http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs
{"data":[],"links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs"},"type":"collection"}
soumya$ curl -X POST http://35.202.251.15:8080/r/projects/1a5/network-diagnostics:8080/v1/logs
{"actions":{},"downloadURL":"","id":"1502319005179725705","links":{"self":"http://35.202.251.15:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1502319005179725705"},"self":"1502319005179725705","state":"creating","type":"log"}
soumya$ curl http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs
{"data":[{"actions":{},"downloadURL":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/static/logs/rancher-logs-1502319005179725705.zip","id":"1502319005179725705","links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs/1502319005179725705"},"self":"1502319005179725705","state":"creating","type":"log"}],"links":{"self":"http://IP:8080/r/projects/1a5/network-diagnostics:8080/v1/logs"},"resourceType":"log","type":"collection"}

```
